privacyidea/lib/policydecorators.py

```diff
@@ -630,29 +624,36 @@ def reset_all_user_tokens(wrapped_function, *args, **kwds):
     :param options: options dictionary containing g.
     :return: None
     """
-    tokenobject_list = args[0]
+    token_objects = args[0]
     options = kwds.get("options") or {}
     g = options.get("g")
     allow_reset = kwds.get("allow_reset_all_tokens")
 
     r = wrapped_function(*args, **kwds)
 
-    toks_avail = [tok for tok in tokenobject_list if tok.get_class_type() not in ['registration']]
+    available_tokens = [tok for tok in token_objects if tok.get_class_type() not in ['registration']]
 
     # A successful authentication was done
-    if r[0] and g and allow_reset and toks_avail:
-        token_owner = kwds.get('user') or toks_avail[0].user
+    if r[0] and g and allow_reset and available_tokens:
+        token_owner = kwds.get('user') or available_tokens[0].user
         reset_all = Match.user(g, scope=SCOPE.AUTH, action=ACTION.RESETALLTOKENS,
                                user_object=token_owner if token_owner else None).policies()
         if reset_all:
             log.debug("Reset failcounter of all tokens of {0!s}".format(
                 token_owner))
-            for tok_obj_reset in toks_avail:
+            for tok_obj_reset in available_tokens:
                 try:
                     tok_obj_reset.reset()
                 except Exception:
-                    log.debug(
-                        "registration token does not exist anymore and "
-                        "cannot be reset.")
+                    log.debug("Registration token does not exist anymore and cannot be reset.")
 
     return r
+
+
+def force_challenge_response(wrapped_function, user_object, passw, options=None):
+    g = options.get("g")
```

Comment:
I think we should check that the `options` dictionary exists like in the other functions.
With this decorator style we can't know if it is actually passed to the functions.